.github/workflows/android-build.yml

```diff
@@ -25,10 +25,14 @@ jobs:
         uses: gradle/gradle-build-action@v2
       - name: Setup Vulkan SDK
         run: |
-          wget -q https://sdk.lunarg.com/sdk/download/1.3.216.0/linux/vulkansdk-linux-x86_64-1.3.216.0.tar.gz
+          wget -q https://sdk.lunarg.com/sdk/download/1.3.250.0/linux/vulkansdk-linux-x86_64-1.3.250.0.tar.gz
```

Comment:
If we are updating the version of the Vulkan SDK, perhaps we should update the one in the windows check as well for consistency?